CHANGELOG.md

```diff
@@ -0,0 +1,53 @@
+# [0.2.0](https://github.com/aerogear/unifiedpush-admin-ui/compare/2.2.0...0.2.0) (2020-09-16)
```

Comment:
CHANGELOG.md is managed automatically by the `version_bump.yml` action, so this should be removed

Reply:
This is the generated file :) The script is running on the wrong branch for testing purposes.